locale/en/docs/guides/nodejs-docker-webapp.md

```diff
@@ -140,15 +145,15 @@ CMD [ "npm", "start" ]
 Your `Dockerfile` should now look like this:
 
 ```docker
-FROM node:boron
+FROM node:carbon
 
 # Create app directory
 WORKDIR /usr/src/app
 
 # Install app dependencies
-COPY package.json .
-# For npm@5 or later, copy package-lock.json as well
-# COPY package.json package-lock.json ./
+COPY package.json package-lock.json ./
```

Comment:
I considered making this a wildcard to account for both cases... but thought this was more straight-forward and consistent what was here before.

The following will _not_ fail if the `package-lock.json` file does not exist (using an older version of `npm` or if `npm install` was not run before `docker build ...`:
```
COPY package.json package-*.json ./
```

Reply:
At this point, `COPY .` should be enough IMHO. No need to explicitly copy `package*.json` again.

Reply:
Just to clarify, are you suggesting to forego the explicit coping of `package*.json` files and simply `COPY . .` _before_ the `npm install` (removing the `# Bundle app source` command)? Resulting in:
```
FROM node:carbon

# Create app directory
WORKDIR /usr/src/app

# Bundle app source
COPY . .

# Install app dependencies
RUN npm install
# If you are building your code for production
# RUN npm install --only=production

EXPOSE 8080
CMD [ "npm", "start" ]
```

Or, replacing the replacing the `COPY package.json package-lock.json ./` with `COPY package*.json .`?

Reply:
Sorry, my mistake, got a bit mixed up in the diff. I meant 

> replacing the replacing the `COPY package.json package-lock.json ./` with `COPY package*.json .`?